tools/tck-api/src/test/scala/org/opencypher/tools/tck/TckTest.scala

```diff
@@ -63,12 +63,17 @@ class TckTest extends AnyFunSpec with Assertions {
           CypherValueRecords.empty
         case ExecQuery if query.contains("foo()") =>
           ExecutionFailed(SYNTAX_ERROR, COMPILE_TIME, UNKNOWN_FUNCTION)
+        // assert that csv path parameter is not overwritten by additional parameters
+        case ExecQuery if query.contains("LOAD CSV") && params.keySet.equals(Set("param", "list")) =>
+          StringRecords(List("res"), List(Map("res" -> "[]"), Map("res" -> "[]")))
```

Comment:
I would test the whole roundtrip (multiple comments):

```suggestion
          StringRecords(List("res"), cvsData.rows.map(r => Map("res" -> r("txt").toString)))
```